cli/src/commands/extensions/mod.rs

```diff
@@ -70,7 +78,13 @@ pub fn add_extensions_subcommands(command: Command<'_>) -> Command<'_> {
 /// Entry point for the `extensions` subcommand.
 pub async fn handle_extensions(matches: &ArgMatches) -> CommandResult {
     match matches.subcommand() {
-        Some(("add", matches)) => handle_add_extension(matches.value_of("PATH").unwrap()).await,
+        Some(("add", matches)) => {
+            handle_add_extension(
+                matches.value_of("PATH").unwrap(),
+                matches.get_one::<bool>("yes").copied().unwrap_or(false),
```

Comment:
Pretty sure there's an easier way to check for flags. I'd have to lookup clap docs myself again but I'm like 90% certain and there's probably a couple examples in our codebase too.

Reply:
I think `is_present` should exist? That's what my memory is of it anyway.